commands/describe.go

```diff
@@ -109,9 +109,20 @@ func runDescribe(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func getFunctionURLs(gateway string, functionName string) (string, string) {
+func getFunctionURLs(gateway string, functionName string, functionNamespace string) (string, string) {
+	var url, asyncURL string
+
 	gateway = strings.TrimRight(gateway, "/")
-	return gateway + "/function/" + functionName, gateway + "/async-function/" + functionName
+
+	if functionNamespace == "" {
```

Comment:
I think you could do away with this repetition in each block. What if you just decide the suffix here and have it as an empty string when there is no namespace? That also means you can do away with the else

Reply:
I like your suggestion--I think the code looks much cleaner now.